### PR TITLE
Add TPS scaling statistics

### PR DESCRIPTION
**Pull request by jokal2**

### CODE REVIEW

File: Statistics/NumberCruncher.cpp [hunk 1]
```diff
@@ -224,7 +224,36 @@ void NumberCruncher::merge_rotation_executor_stats(QList<QList<ExecutorOutcome*>
...
+
+    double base_value_tps = tps_per_option.take(SimOption::Name::NoScale);
+
+    QMap<SimOption::Name, double>::const_iterator tps_it = tps_per_option.constBegin();
+    while (tps_it != tps_per_option.constEnd()) {
+      double absolute_diff = tps_it.value() - base_value_tps;
+      double relative_diff = absolute_diff / base_value_tps;
+
+      double standard_deviation = get_standard_deviation_for_option(tps_it.key());
+      double confidence_interval = get_confidence_interval_for_option(tps_it.key(), standard_deviation);
```
Comment (timhul) [inline Statistics/NumberCruncher.cpp:249]: Indentation is lower than expected in this project (and as formatted by clang-format).

File: main.cpp [hunk 2]
```diff
@@ -71,7 +71,8 @@ int main(int argc, char* argv[]) {
     context->setContextProperty("rotationExecutorBreakdownModel", gui_control->get_rotation_executor_model());
     context->setContextProperty("rotationExecutorListModel", gui_control->get_rotation_executor_list_model());
     context->setContextProperty("simScaleModel", gui_control->get_sim_scale_model());
-    context->setContextProperty("scaleResultModel", gui_control->get_scale_result_model());
```
Comment (timhul) [inline main.cpp:74]: There are still some references to scaleResultModel in StatisticsScaleResultSorting.qml which causes errors. You'll probably need to pass the model it should use for its operations.

Comment (jokal2) [reply main.cpp:74]: hmm. Wanna commandeer this commit? Not sure how to parameterize that QML... type? to pass down the right `ScaleResultModel`

Comment (timhul) [reply main.cpp:74]: I'll accept a QML error for now and fix it later as it is fairly minor.

**Pull request merged**